tests/api/client/login_test.c

```diff
@@ -9,12 +9,27 @@ static void test_login(void)
 {
     int rc;
 
+    rc = hive_client_logout(test_ctx.client);
```

Comment:
"test_login" should be the flow sequence of :
1. hive_client_login()
2. hive_cllent_loug()